proto/zepben/protobuf/ns/data/change-events.proto

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2021 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.nm";
+option csharp_namespace = "Zepben.Protobuf.NM";
+package zepben.protobuf.nm;
+
+import "google/protobuf/timestamp.proto";
+import "zepben/protobuf/cim/iec61970/base/core/PhaseCode.proto";
+
+/**
+ * An event to apply to the current state of the network.
+ */
+message CurrentStateEvent {
+
+    /**
+     * An identifier of this event. This must be unique across requests to allow detection of
+     * duplicates when requesting events via dates vs those streamed via live updates.
+     */
+    string eventId = 1;
+
+    /**
+     * The timestamp when the event occurred.
+     */
+    google.protobuf.Timestamp timestamp = 2;
+
+    /**
+     * The type of event that occurred.
+     */
+    oneof event {
+        SwitchStateEvent switch = 3;
+        AddCutEvent addCut = 4;
+        RemoveCutEvent removeCut = 5;
+        AddJumperEvent addJumper = 6;
+        RemoveJumperEvent removeJumper = 7;
```

Comment:
i reckon this is probably correct for cuts and jumpers, but i don't have enough information to confirm that yet. At the very least the events are correct, it'll just be a question of what data is captured in them which we won't find out until later.

Reply:
Yeah was only meant to be placeholders with proper messages to be done as a future pass when we have details on what is required